### PR TITLE
[CodingStyle] Handle ClassConstFetch on RemoveUnusedAliasRector

### PR DESCRIPTION
**Pull request by samsonasik**
After PR https://github.com/rectorphp/rector-src/pull/784 , I think ClassConstFetch should be removed too if there is no conflict with use alias, it only make sense to have alias only if has conflict name in use statement.

Ref for original issue: https://github.com/rectorphp/rector/issues/5687
Ref for original PR that introduce it: https://github.com/rectorphp/rector/pull/5691

ref https://3v4l.org/H4lrmT for demo that it actually same to use alias or not.

### CODE REVIEW

**Comment by samsonasik** [comment]
All checks have passed 🎉 @TomasVotruba it is ready for review.

File: rules/CodingStyle/Rector/Use_/RemoveUnusedAliasRector.php [hunk 1]
```diff
@@ -183,28 +182,19 @@ private function shouldSkip(Use_ $use, Name $name, string $lastName, string $ali
...
+            $originalName = $node->getAttribute(AttributeKey::ORIGINAL_NAME);
+            if (! $originalName instanceof Name) {
                 return false;
             }
 
-            return $node->class->toString() === $name->toString();
+            $loweredOriginalName = strtolower($originalName->toString());
+            $loweredOriginalNameNamespace = Strings::before($loweredOriginalName, '\\');
+            return $loweredAliasName === $loweredOriginalNameNamespace;
         });
```
Comment (TomasVotruba) [inline rules/CodingStyle/Rector/Use_/RemoveUnusedAliasRector.php]: Could you extract this to a XFinder class? It seems it could be re-used elsewhere

Comment (samsonasik) [reply rules/CodingStyle/Rector/Use_/RemoveUnusedAliasRector.php]: implemented https://github.com/rectorphp/rector-src/pull/785/commits/d0dc5061fc6bc95320d016ec45c1655e50ccf39d, add FullyQualifiedFromUseFinder service for it

**Comment by samsonasik** [comment]
All checks have passed 🎉 @TomasVotruba I think it is ready.

**Comment by TomasVotruba** [comment]
Thank you :+1:

**Pull request merged**